apps/cloudappclient/app.js

```diff
@@ -61,6 +61,16 @@ module.exports = activity => {
           logger.log(`${skill.appId}: an error occur when destroy media ${err}`)
         })
     }
+    // continuous dialogue only for cloudappclient.
+    if (sos.skills.length > 1 && property.get('persist.sys.pickupswitch') === 'open') {
```

Comment:
It should pick up only when last skill is cut form. How do we determine last skill is cut form?

Reply:
The value of skills is always be [scene, cut], so we can judging by the length of the array on lifecycle exit.